src/app.module.ts

```diff
@@ -1,13 +1,28 @@
 import { Module } from '@nestjs/common';
-
+import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 
 import { CartModule } from './cart/cart.module';
 import { AuthModule } from './auth/auth.module';
 import { OrderModule } from './order/order.module';
+import * as process from 'process';
 
 @Module({
   imports: [
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: process.env.PG_HOST,
+      port: Number(process.env.PG_PORT),
+      username: process.env.PG_USER,
+      password: process.env.PG_PASS,
+      database: process.env.PG_DATABASE_NAME,
+      entities: [__dirname + '/**/entity/*.js'],
+      synchronize: false, // this prop each time try to create a new table so if table is exist will be error
```

Comment:
Your comment is not actually fully truth. It is synchronizing your database entities changes, if there is changes in it, then it will try to update it, and there might be errors like not-null constraint.